Source/WebCore/css/calc/CSSCalcTree.h

```diff
@@ -1204,6 +1224,8 @@ OP_TUPLE_LIKE_CONFORMANCE(Abs, 1);
 OP_TUPLE_LIKE_CONFORMANCE(Sign, 1);
 OP_TUPLE_LIKE_CONFORMANCE(Progress, 3);
 OP_TUPLE_LIKE_CONFORMANCE(Anchor, 0);
+// TODO: fix this.
```

Comment:
TODO really applies to Anchor too.

Reply:
I'll move it up and maybe create a bugzilla

Reply:
https://bugs.webkit.org/show_bug.cgi?id=280798